core/templates/dev/head/components/forms/schema_editors/schema_based_html_editor_directive.html

```diff
@@ -1,7 +1,7 @@
 <!-- label-for-focus-target feature is removed since CKEditor has a
 focusManager to manage the focus activity automatically. -->
 <ck-editor-rte ng-if="!isDisabled()" ng-model="$parent.localValue"
-               ui-config="uiConfig()">
+               ui-config="uiConfig()" style="word-break: break-all;">
```

Comment:
Do you really need `word-break` here as well ? (I'm not sure but I myself just don't want to add here)

Reply:
@bansalnitish And not only interactions, we will have to specify word break for all places where editior is used explicitly if we don't specify it here.

The introduction looked like this before.
![image](https://user-images.githubusercontent.com/32777813/50049368-5b070c00-010a-11e9-8747-edc7cade5dc2.png)
Now it looks like this
![image](https://user-images.githubusercontent.com/32777813/50049374-740fbd00-010a-11e9-9fed-2749ee0c7448.png)
Thanks!

Reply:
Is it possible to add this as a global style for the ck-editor-rte tag? Or do we not want this for all instances of the RTE?

Reply:
I think there is no such instance of rte where this style is not to be followed.
Kindly tell me your views on it so that I can make it global. Currently, I just saw that there are other places where the same error is occuring. Such as feedback tab. So should I undo the changes I did in this PR regarding this paticular problem and make a new issue for it? Because this issue seems pretty vast.

Reply:
I think we should have the word-break css style applied to the whole RTE itself (probably, we should place it in oppia.css). Instead of having it in multiple locations (It is easy to miss some of the places it is being used). By making it global we also make sure that any newly used locations of the RTE follows the same rules. I think we should instead do the global thing directly. Does this sound good?

Reply:
Alright. Will make the changes in css itself. Thanks! @nithusha21

Reply:
@nithusha21 The problem is that there isn't only 1 class for editors and editor content holders. There are a number of classes for that. So making changes in css will take some time. I'd rather suggest you to merge this PR, and then we can work on it making a seperate issue. What do you think about it?

Reply:
@nithusha21 Would you like me to undo the changes to editor and editor content holders in this PR, make a new issue, and work on all the editor classes and their content holders?

Reply:
Hmmm. I see. I was under the impression that it was a single class throughout. Also I just went through the code involved, and it looks like the style tag was used on many different components as you said. 

Just to confirm one thing, All the editors would be ck-editor-rte element itself right?

I agree the display for the content will be different, and I think the cleanest way to do this global style would be to use a common class for all of these tags, and add common styles in oppia.css. 

If we actually defer this to another PR, I think that PR would revert a lot of the style changes here, and add it elsewhere. So I think it would make sense to do it here itself? (As anyway these changes would be removed). What do you think? However, I think if we do the global approach, we would need to test it carefully, and make sure all RTEs continue to work properly.

I want to commend you on doing a great job of identifying all the RTEs and carefully testing their behavior. Well done!

Reply:
@nithusha21 Thanks! I don't have any problem in working on the changes in this issue itself. It might take a few days to carefully make the changes and test them.